Documentation/Functions/HtmlparserTags.rst

```diff
@@ -108,10 +108,8 @@ fixAttrib.[attribute].upper
 
     :Data type: :ref:`data-type-boolean`
 
-    If true, the value is passed through the
-    respective PHP-function.
-
-    .. TODO: There is not such PHP function, probably strtoupper?
+    If true, the value is passed through the PHP function
+    `strtoupper() <https://www.php.net/manual/en/function.strtoupper.php>`__.
```

Comment:
Wonder if we might link to mb_stroupper instead? My guess is that function is used instead of the multibyte-unaware one...? I guess that's the reason why it was written as "respective PHP-function", because it depends on the existence of the php-mbstring extension (which is recommended)...?

Reply:
Look at the link I provided in the commit message: Core uses strtoupper() and strtolower(), so this is the way it (currently) works.

Reply:
True. I was completely sure that the core used mb_* functions through a layer like GeneralUtility. Thanks for clearing this up for me.